src/frame_compare/services/report/assets/viewer.css

```
@@ -808,86 +808,249 @@ select optgroup {
+    width: 1px;
+    height: 24px;
 }
 
-.rv-inspection-point > span::after {
-    top: -7px;
-    bottom: -7px;
-    left: 50%;
-    border-left: 1px solid var(--annotation);
+.rv-lens {
+    --lens-size: 240px;
+    position: absolute;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)